src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx

```diff
@@ -148,29 +150,27 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
           </form>
         </DialogContent>
         <DialogActions>
-          <Button
-            color="success"
+          <NERFailButton
+            className={'ml-3'}
             variant="contained"
```

Comment:
I dont know if we need this cant tell the difference when Its deleted

Reply:
Which part do we not need?

Reply:
The classname